### PR TITLE
Initial PySide2 support

### PR DESCRIPTION
**Pull request by jschueller**
I've not tried to dig into the new/missing classes in pyside2 ; this mostly mimic pyside1, but it's a start.

### CODE REVIEW

File: AnyQt/QtCore.py [hunk 1]
```diff
@@ -298,6 +298,21 @@
     del _QtCore, _QtGui, _major, _minor, _micro
     # Known to be in PyQt4 but missing in PySide:
     #     Q_ARG, Q_CLASSINFO, Q_ENUMS, Q_FLAGS, Q_RETURN_ARG, ...
+elif _api.USED_API == _api.QT_API_PYSIDE2:
+    from PySide2 import QtCore as _QtCore
+    globals().update(
+        {name: getattr(_QtCore, name)
+         for name in __Qt4_QtCore if hasattr(_QtCore, name)}
```
Comment (ales-erjavec) [inline AnyQt/QtCore.py]: This should probably import full namespace from PySide2.QtCore. Currently the `QAbstractProxyModel`, `QItemSelection`, ... are not re-exported (at the least it should export the combined `__Qt4_QtCore + __Qt4_QtGui` names).  Note also that `QStringListModel` still seems to be in PySide2.QtGui.

File: AnyQt/QtWidgets.py [hunk 2]
```diff
@@ -326,6 +326,14 @@ def _QWidget_grab(self, rect=__QtCore.QRect(0, 0, -1, -1)):
     QWidget.grab = _QWidget_grab
     del _QtGui, __QtCore
 
+elif _api.USED_API == _api.QT_API_PYSIDE2:
+    from PySide2 import QtWidgets as _QtWidgets
+    globals().update(
+        {name: getattr(_QtWidgets, name)
+         for name in __Qt4_QtGui if hasattr(_QtWidgets, name)}
```
Comment (ales-erjavec) [inline AnyQt/QtWidgets.py]: Same here. Import the full namespace.

File: AnyQt/QtGui.py [hunk 3]
```diff
@@ -152,8 +152,15 @@
     del _QtGui
     # Known to be present in PyQt4 but not in PySide:
     #   QGlyphRun, QRawFont, QStaticText, QTextDocumentWriter
+elif _api.USED_API == _api.QT_API_PYSIDE2:
+    from PySide2 import QtGui as _QtGui
+    globals().update(
+        {name: getattr(_QtGui, name)
+         for name in __Qt4_QtGui if hasattr(_QtGui, name)}
+    )
```
Comment (ales-erjavec) [inline AnyQt/QtGui.py]: Same here. Import the full namespace.

**Review by ales-erjavec — CHANGES_REQUESTED** [review]

File: AnyQt/QtGui.py (hunk 3 above)
Comment (jschueller) [reply AnyQt/QtGui.py]: ok

File: AnyQt/QtCore.py (hunk 1 above)
Comment (jschueller) [reply AnyQt/QtCore.py]: I reported the QStringListModel issue upstream, I'm testing a patch so we dont have to fix it here as no stable version is out yet:
https://bugreports.qt.io/projects/PYSIDE/issues/PYSIDE-614

**Comment by jschueller** [comment]
I also added qml, qtquick qwebsockets as they are common with pyqt5.
I mentioned Qt5-only in their doc.

File: AnyQt/QtQuick.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+from . import _api
+
+if _api.USED_API == _api.QT_API_PYQT5:
+    from PyQt5.QtQuick import *
+if _api.USED_API == _api.QT_API_PYSIDE2:
```
Comment (ales-erjavec) [inline AnyQt/QtQuick.py]: This needs to be an `elif:`

**Review by ales-erjavec — CHANGES_REQUESTED** [review]

File: AnyQt/QtQuick.py (hunk 4 above)
Comment (jschueller) [reply AnyQt/QtQuick.py]: Ok, fixed

**Review by ales-erjavec — APPROVED** [review]

**Pull request merged**